diff -up aMule-2.1.3/src/ExternalConnector.h.multiple aMule-2.1.3/src/ExternalConnector.h
--- aMule-2.1.3/src/ExternalConnector.h.multiple	2008-04-08 15:33:04.000000000 +0200
+++ aMule-2.1.3/src/ExternalConnector.h	2008-04-08 15:38:22.000000000 +0200
@@ -143,7 +143,7 @@ public:
 	bool Parse_Command(const wxString& buffer);
 	void GetCommand(const wxString &prompt, char* buffer, size_t buffer_size);
 	const CECPacket *SendRecvMsg_v2(const CECPacket *request) { return m_ECClient->SendRecvPacket(request); }
-	void ConnectAndRun(const wxString &ProgName, const wxString& ProgName);
+	void ConnectAndRun(const wxString &ProgName, const wxString& ProgVersion);
 	void ShowGreet();
 
 	//
